### PR TITLE
Add links to the tutorial and highlight the Windows requirement

### PR DESCRIPTION
**Pull request by skullydazed**

### CODE REVIEW

**Comment by zvecr** [comment]
Not sure I like the `The one-click installer for MSYS2 + QMK CLI on Windows` tagline. Maybe there just needs to be more emphasis in the homepage, guide, etc. Or maybe we ignore the MSYS2 part and go with `The windows one-click installer for QMK CLI`?

**Comment by skullydazed** [comment]
I'm ok with that, my main objection is that it's not very clear this is a windows thing. Want to make a suggestion on the PR for the wording you like?

**Comment by zvecr** [comment]
> main objection is that it's not very clear this is a windows thing

Thats true, I guess i had assumed that the MSYS context would follow. But for the target market, lets be more explicit. I can make the suggestion, but its also in the top level README so cannot apply one there.

File: docs/guide.md [hunk 1]
```diff
@@ -11,6 +11,10 @@
 1. Profit!  
 <img :src="$withBase('/finish.png')" alt="built firmware example" width="70%">
 
+::: tip Next Steps
+The [QMK Tutorial](https://docs.qmk.fm/#/newbs_building_firmware) will walk you through creating a keymap for your keyboard.
+:::
+
```
Comment (zvecr) [inline docs/guide.md:17]: Visual style here of two tip boxes is a bit cluttered, maybe something like:

```suggestion
## Next Steps
The [QMK Tutorial](https://docs.qmk.fm/#/newbs_building_firmware) will walk you through creating a keymap for your keyboard.
:::

```

**Review by zvecr — CHANGES_REQUESTED** [review]
config.js description and beta warning need to be copied to top level readme, but I can do that after if required.

**Review by zvecr — APPROVED** [review]

**Pull request merged**